### DISCUSSION TITLE
How to tell Uno also detect class name from other attribute

### DISCUSSION THREAD

**Discussion by tammai-bigin** [unocss/unocss#1253] [Q&A]
Hi,
I am using Uno with Vue components and facing a problem that I don't know how to make Uno work when I use other attribute like `body-class="p-4"`. Is it possible?

Thank you all!

**Comment by chu121su12** [comment]
If you use `body-class=` a lot, you may want to use dummy variant to match & strip `body-class-` and treat it as attributify. Something [like this](https://unocss.antfu.me/play/?html=DwEwlgbgBARg9iAngWgMYBsCGBnbBeAIgAdkAWAgPgEIAoIA&config=JYWwDg9gTgLgBAbwFBzgEwKYDNgDsMDCEuOA5gDQpxhQYDOGMAqrhJajfYwIIwxTAARgFcYwLAE9KAXzhYoEEHADkw1gGM6dZUiQYAHpFjpsAQ2EAbeJhz4iJYKQAUyDrQYw6ALjgBtKm5czKxOAJTsqNTuPHwCImKSYREAuhEAbqYCpriePv6RiAGRIKYw6gAWTiVl5RhQoYUFBerEdPDVFXAAvHAdtVAAdH1OAPQAeoIQaBIAtOoWplozI6FFBeJwVaUVDa5NBbQwwlC4jftNfXU%2Bl4N0FsDqGFs1vgAMyQMWGLikMOXhaya0kBqGB%2B2kEVBKRkqyAA&options=N4IgzgLgTglgxhEAuaBXApgXyA).
